src/main/scala/fucc/cipher/instance.scala

```diff
@@ -0,0 +1,32 @@
+package fucc.cipher
+
+//import cats.instances.either
+import cats.syntax.either._
+
+import java.security.SecureRandom
+import javax.crypto.spec.IvParameterSpec
+import javax.crypto.{SecretKey, Cipher => JCipher}
+
+object instance {
+
+  val `DES/ECB/PKCS5Padding` = new Cipher[DES, ECB, `PKCS5Padding`] {
+
+    def generator = JCipher.getInstance(s"${DES.tag.algorithm}/${ECB.tag.algorithm}/${`PKCS5Padding`.tag.algorithm}")
+    def getIV(): Array[Byte] = List(0,0,0,0,0,0,0,0).map(_.toByte).toArray
+
+    override def encrypt(clearText: ClearText)(implicit key: SecretKey): Either[CipherError, CipherText] = {
```

Comment:
You have to be careful here. IV is an initialization vector, and depends on your cipher instance.

getIV depends on the cipher itself. You would call it on an instance of Cipher, 

In general, make getIV() take the cipher instance.